### DISCUSSION TITLE
Changelog?

### DISCUSSION THREAD

**Discussion by PLWpl** [taoqf/node-html-parser#102] [Q&A] [answered]
Hello,

When I added node-html-parser as a dependency, it had a version of 1.2.19. Now, 4 months later, it's at 3.0.2, but I can't find any changelog that would tell me if there are any breaking changes.

Do you follow standard convention (major(breaking).minor.bugfix)? Does v3 mean it's got some breaking changes compared to v1? What are these, where can I find them?

Thanks a lot!

//edit//
So, I risked it and updated anyway, and it seems to be working - but my question, just for future updates' sake, stands :)

**Comment by taoqf** [comment, marked as answer]
I'm so sorry for that.
v2: parse option changed, user who use custom options will be affected.
v3: support css3 selector.
and fixed some issues.
I hope you will open issues if you get trouble with 3.x, and we will make this lib better together.

**Reply by PLWpl** [reply]
Thanks a lot for your reply :) For now I see no issues, but of course, if any appear I'll be happy to contribute.

**Comment by taoqf** [comment]
https://github.com/taoqf/node-html-parser/releases
